substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/hub/DynamicHub.java

```diff
@@ -755,6 +756,12 @@ private boolean isAnonymousClass() {
         }
     }
 
+    @Substitute
+    @TargetElement(onlyWith = JDK15OrLater.class)
+    public boolean isHidden() {
+        throw VMError.shouldNotReachHere();
```

Comment:
We need to immediately have a complete implementation. So you need to preserve this information from the image generator in a flag in `DynamicHub` so that you can return it here.

Reply:
Complete Hidden Class support is most likely going to take a lot of additional effort.  Creating Hidden classes is implemented in Hotspot and the hidden attribute is extracted via a native call to Hotspot.  My goal with this fix is to just get past these existing problems with JDK 15 and allow native-image to be used in the mean time.

Reply:
Why would getting the information be difficult? You only need to call the `isHidden` method during image generation and store it in `DynamicHub`. `SVMHost.createHub` already has the `javaClass`.
You need to call `isHidden` via reflection since it doesn't exist in prior JDKs. But that's OK.

Reply:
I can do that but there are other changes throughout the JDK that added support for isHidden like reflection and methodhandles.  These need to be looked at carefully.  Isn't it better to get an error if someone is trying to reference the isHidden attribute rather than to just report that is was or wasn't hidden during native-image generation?  I can go either way.

Reply:
Preserving the information from the image builder is the correct thing to do. There are never any new classes created at image run time, and the "hidden" attribute never changes at run time.

Reply:
Ok, I'll add a reflected call to get the value and store it in a local DynamicHub field.

Reply:
Done.